lib/bootstrap_form/form_builder.rb

```diff
@@ -52,7 +52,9 @@ def initialize(object_name, object, template, options)
       define_method(with_method_name) do |name, options = {}, html_options = {}|
         prevent_prepend_and_append!(options)
         form_group_builder(name, options, html_options) do
-          content_tag(:div, send(without_method_name, name, options, html_options), class: control_specific_class(method_name))
+          html_class = control_specific_class(method_name)
+          html_class = "#{html_class} form-inline" unless options[:skip_inline]
```

Comment:
I suspect that you will need to be more specific here about when to add `form-inline`. I believe you only want to add it when the `form_layout` is `:horizontal`.